tests/fv3_conf/fv3_slurm.IN_noaacloud

```diff
@@ -13,11 +13,9 @@ echo -n " $( date +%s )," >  job_timestamp.txt
 
 set +x
 MACHINE_ID=noaacloud
-#module use $( pwd -P )
-#module load modules.fv3
-#module list
-module use /lustre/ufs-weather-model/modulefiles
-module load ufs_cloud.intel
+module use $( pwd -P )
+module load stack-intel/2021.3.0  stack-intel-oneapi-mpi/2021.3.0
+module load ufs-weather-model-env/unified-dev
```

Comment:
i am not sure you should load the modules like that. it may result in inconsistencies. for one: loading ufs_noaacloud.intel as you have it results in a different set of modules than loading it as in your recent commit. see the comparison screencaps below. secondly, the MODULEPATH needs to be prepended before the spack-stack meta-modules (stack-intel and stack-intel-oneapi-mpi) can be properly loaded -- it may work sometimes but not others unless the paths are properly set. 

why is the noaacloud module not getting copied over to the RT dir as modules.fv3 and loaded properly? curious about that error. 

![Screen Shot 2023-04-18 at 7 11 28 AM](https://user-images.githubusercontent.com/43379611/232804552-bd2f4cfb-f12f-435d-9a8e-299f3a98bf42.png)
![Screen Shot 2023-04-18 at 7 11 54 AM](https://user-images.githubusercontent.com/43379611/232804572-3d21a441-775a-49df-abbf-1f7d39dddb90.png)

Reply:
Hey @ulmononian I'm going to add the MODULEPATH here in a second, I left it out by mistake. The error I was receiving regarding modules.fv3, was that "module modules.fv3 could not be found."